pkgs/development/python-modules/sphinx-intl/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, Babel
+, click
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-intl";
+  version = "2.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1d1q0sanjp4nkfvhsxi75zf3xjyyi8nzxvl3v7l0jy9ld70nwnmj";
+  };
+
+  propagatedBuildInputs = [ Babel click sphinx ];
+
+  meta = with stdenv.lib; {
+    description = "Translation support for Sphinx";
+    homepage = "https://github.com/sphinx-doc/sphinx-intl";
+    license = licenses.bsd2;
+    maintainers = with lib.maintainers; [ euandreh ];
```

`lib` is not required here, as its already in scope due to the `with stdenv.lib;` above.
```suggestion
    maintainers = with maintainers; [ euandreh ];
```